lib/flight_cert/cli.rb

```diff
@@ -82,6 +82,15 @@ def self.create_command(name, args_str = '')
       c.slop.bool '--config-only', 'Only update the internal configuration, skips certificate generation'
     end
 
+    create_command 'cert-install' do |c|
+      c.summary = 'Use a self-generated SSL certificate'
+      c.description = <<~DESC
+        Use an SSL certificate that has been generated by the user, independent of Flight Cert.
+      DESC
+      c.slop.string '--key', 'Specify the path to the key file', required: true
+      c.slop.string '--fullchain', 'Specify the path to the fullchain file', required: true
```

Comment:
Generally, I'm not a fan of required options.  One of the reasons is that it provides odd interactions such as the following:

Viewing the help text via `help COMMAND` works with required options.

```
[vagrant@chead1 flight-cert]$ /opt/flight/bin/ruby ./bin/cert help cert-install 
  SYNOPSIS:

    bin/cert cert-install 

  DESCRIPTION:

    Use an SSL certificate that has been generated by the user, independent of Flight Cert.
    

  OPTIONS:
	
    --key KEY
        Specify the path to the key file
  
    --fullchain FULLCHAIN
        Specify the path to the fullchain file
```

but viewing the help text with `COMMAND --help` does not.

```
[vagrant@chead1 flight-cert]$ /opt/flight/bin/ruby ./bin/cert cert-install --help
bin/cert: missing required option `--key'
```

I'd sooner have the command be `cert-install KEY_PATH FULL_CHAIN_PATH` and require the user to remember the order than to have a broken `--help` flag.